git-hooks/util/formatting.js

```diff
@@ -0,0 +1,80 @@
+// See http://bit.ly/2ORJ0x2 (\x1b sequence)
```

Comment:
Please see comment above concerning `semicolons`. 

I recommend adding the semicolons.

Reply:
Semicolons added.